src/derivatives/atlas.md

```diff
@@ -55,6 +53,7 @@ several sessions and/or subjects, the derivatives-specific
         [cohort-<label>/]
            [<datatype>/]
                tpl-<label>_<source_entities>[_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
+               atlas-<label>_<source_entities>[_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
```

Comment:
```suggestion
```

What is the use case? I'd like to see actual examples of this to see in practice. Also, I don't think you want to duplicate atlas in here.